Userland/Libraries/LibGL/Vertex.cpp

```diff
@@ -194,6 +194,13 @@ void GLContext::gl_draw_elements(GLenum mode, GLsizei count, GLenum type, void c
     gl_end();
 }
 
+void GLContext::gl_normal_double(GLdouble nx, GLdouble ny, GLdouble nz)
```

Comment:
Instead of adding this new function, you can call the existing `gl_normal` function by casting to float first.